scripts/pre_commit_linter.py

```diff
@@ -788,6 +788,31 @@ def handle_starttag(self, tag, attrs):
         expected_indentation = self.indentation_level * self.indentation_width
         tag_line = self.file_lines[line_number - 1].lstrip()
         opening_tag = '<' + tag
+
+        # Check the indentation for content of style tag.
+        if tag_line.startswith(opening_tag) and tag == 'style':
+            next_line_number = line_number + 1
+            next_line = self.file_lines[next_line_number - 1]
```

Comment:
Swap this line with the previous line otherwise you are doing an unnecessary operation.

Reply:
I can't get it. I am using next_line_number to print an error.

Reply:
You are adding 1 in the previous line and then subtracting it immediately in the next line, thus canceling out the +1 from the line before.

Reply:
Done.